config/environments/test.rb

```diff
@@ -43,6 +43,15 @@
     Bullet.add_whitelist :type => :n_plus_one_query,
                          :class_name => 'BusinessUnit',
                          :association => :correspondence_types
+    Bullet.add_whitelist :type => :n_plus_one_query,
+                         :class_name => 'BusinessGroup',
+                         :association => :moved_to_unit
+    Bullet.add_whitelist :type => :n_plus_one_query,
+                         :class_name => 'Directorate',
+                         :association => :moved_to_unit
+    Bullet.add_whitelist :type => :n_plus_one_query,
+                         :class_name => 'BusinessUnit',
+                         :association => :moved_to_unit
```

Comment:
These here to fix exception thrown by Bullet - flagging an N+1 issue because of a self-referential table inheritance field. I think it's a red herring - in any event was unable to re-jig the code to Bullet's satisfaction so added to whitelist - see thread here https://mojdt.slack.com/archives/CC3A275NU/p1572363343010300

```
(byebug) page.body
"Bullet::Notification::UnoptimizedQueryError at /stats\n=====================================================\n\n> 
user: iananderson\nPOST /stats\n
USE eager loading detected\n  BusinessUnit => [:moved_to_unit]\n  
Add to your finder: :includes => [:moved_to_unit]\nCall stack\n  /Users/iananderson/dev/moj/cts/app/models/team.rb:168:in `moved'\n  
```